crates/block-producer/src/store/mod.rs

```diff
@@ -53,15 +50,6 @@ pub trait Store: ApplyBlock {
         produced_nullifiers: impl Iterator<Item = &Nullifier> + Send,
         notes: impl Iterator<Item = &NoteId> + Send,
     ) -> Result<BlockInputs, BlockInputsError>;
-
-    /// Returns note authentication information for the set of specified notes.
-    ///
-    /// If authentication info for a note does not exist in the store, the note is omitted
-    /// from the returned set of notes.
-    async fn get_note_authentication_info(
-        &self,
-        notes: impl Iterator<Item = &NoteId> + Send,
-    ) -> Result<NoteAuthenticationInfo, NotePathsError>;
```

Comment:
I think we might need to bring this back in the future. Do we have an issue for this?

Separately (and also for a different PR), I'm sure we need to have `ApplyBlock` as a separate trait. Maybe worth moving `apply_block()` method to the `Store` trait?

Reply:
#565 and #566